src/dotnet/APIView/APIView/Model/StructuredTokenModel.cs

```diff
@@ -35,6 +37,52 @@ public enum StructuredTokenKind
         ParameterSeparator = 4
     }
 
+    public enum DiffKind
+    {
+        NoneDiff = 0,
+        Unchanged = 1, // Unchanged means the top level node is the same, the children could still contain diffs.
```

Comment:
Can we rename this to ChangedSubTree is this enum is set only when current level is same but somewhere down the line there is a diff? I assume enum value will be NoneDiff if current level as well as it's entire child trees are not changed.

Enum to show following:
Current level has addition
Current level has deletion
Current level is not modified but current level has a modified child tree
Current level as well as it's entire subtree is not modified.

Reply:
This is way more complicated to understand.